README.rst

```diff
@@ -85,6 +85,32 @@ directory you can do this::
 If there's not output. It worked. Check how it edited your
 requirements files.
 
+Filtering requirements by Python version
```

Comment:
"Filtering **hashes** by Python version"

To me "requirements" is another word for the "packages". Ultimately what we're meddling with is the listed hashes. If someone wants to hashin Flask, Django and Tornado, we don't filter on that.